### DISCUSSION TITLE
How to customize SYMMETRY FUNCTIONS ?

### DISCUSSION THREAD

**Discussion by wangjiuning123** [MarsalekGroup/aml#30] [Q&A] [answered]
![image](https://github.com/MarsalekGroup/aml/assets/72865425/7201a496-6480-49e8-897f-a94d078e2ea0)
Please tell me How to pass the parameters to symmetry functions for own use?

**Comment by beckobert** [comment, marked as answer]
Since this question concerns the N2P2 package, I advice to check out the official N2P2 documentation, in particular [this section](https://compphysvienna.github.io/n2p2/topics/keywords.html#symfunction-short)

AML will only write symmetry functions if the `input.nn` file contains the formatting instructions

> {acsf:}

so simply delete this line and insert your own preferred symmetry functions instead.
